articles/test-runner-plugins.md

```diff
@@ -0,0 +1,233 @@
+<!--
+parent: 'Developer Guide'
+skip : true
+tags:
+    - 'Frontend'
+    - 'Plugin'
+    - 'JavaScript'
+    - 'Test Runner'
+-->
+Enhance the user experience with Test Plugins
+=============================================
+
+> One of the best option to improve, customize the test taker experience is using _Test Runner Plugins_.
+A plugin can be anything that hook the test experience : some tools to help the test taker like a calculator, progress information, etc.
```

Comment:
..that hooks into the test experience: some tools to help the test taker such as a calculator